README.md

```diff
@@ -263,7 +263,7 @@ plugins or client-side software.
 
 ## Setting the content type
 
-As of v0.10.0, the `mime-types` gem is a runtime dependency and the content type is set automatically.
+As of v0.11.0, the `mime-types` gem is a runtime dependency and the content type is set automatically.
```

Comment:
I just checked and `mime-types` is still a runtime dependency. So I'd say either leave `0.11` there or completely remove the reference to a specific version.